orchestra/tests/test_views.py

```diff
@@ -131,3 +150,81 @@ def test_time_entries_post_data_invalid(self, mock_func, mock_logger):
                                         data=json.dumps(self.time_entry_data),
                                         content_type='application/json')
         self._verify_bad_request(resp, "['Error']")
+
+
+class TimerEndpointTests(EndpointTests):
+
+    def setUp(self):
+        super().setUp()
+        user = UserFactory(username='test_user')
+        self.worker = WorkerFactory(user=user)
+        self.request_client = RequestClient()
+        self.request_client.login(username=self.worker.user.username,
+                                  password='defaultpassword')
+        self.time = timezone.now()
+
+    @patch('orchestra.views.time_tracking.start_timer')
+    def test_start_timer(self, mock_start):
+        timer = TaskTimer(worker=self.worker, start_time=self.time)
```

Comment:
if there is a start_time already set, shouldn't we get an exception saying the timer is already running?